extra/3RD_PARTY_QUICK_START.md

```diff
@@ -9,48 +9,55 @@ Take a look at some of the real-life examples first:
 - https://github.com/highlightjs/highlightjs-cypher
 - https://github.com/highlightjs/highlightjs-robots-txt
 
-Basically:
+- Checkout a clone of the core repo [highlight-js](https://github.com/highlightjs/highlightjs) from GitHub.
+- Read [Language contributors checklist](https://highlightjs.readthedocs.io/en/latest/language-contribution.html).
 
-- Checkout highlight-js from github...
-- 3rd party languages are placed into the `extra` directory
+## Create a repo
 
-So if you had a `xyz` language you'd create an `extra/xyz` folder, and that would be your language module. All paths below are relative to that.
+Each language is developed in its own repo. This helps keep language definitions and maintainers independent of the core work.
+Determine if you will host the repository yourself or you want it to be part of the [highlightjs organization on GitHub](https://github.com/highlightjs).
```

Comment:
This should be reworded to encourage people to start with their own repo - we can migrate it to our organization later if the user requests.  What happens when we let them decide first is we end up having a few people who never finish their grammar and then we have dead repos in our organization for a long time - and it's hard to decide when they should be removed.  Just awkward.

Would be much nicer to have someone send a link to their repo all shiney and ready to go and then if everything looks good it can be transferred or forked, etc... I think that's the sequencing we'd like to head towards.

CC @allejo

Reply:
I agree with Josh. Having a complete and functional grammar would at least allow hljs maintainers to approve PRs should it be abandoned. But if it's incomplete, then it'd be up to the maintainers to complete it.

Reply:
including a reference to the template should make this easier